Framework/CurveFitting/test/Functions/ConvolutionTest.h

```diff
@@ -409,87 +393,6 @@ class ConvolutionTest : public CxxTest::TestSuite {
                       1e-10);
     }
   }
-
-  void testForCategories() {
-    Convolution forCat;
-    const std::vector<std::string> categories = forCat.categories();
-    TS_ASSERT(categories.size() == 1);
-    TS_ASSERT(categories[0] == "General");
-  }
-
```

Comment:
Thanks for removing all unnecessary stuff, including includes. But shouldn't we keep the `testForCategories()`?